src/App.js

```diff
@@ -1,24 +1,26 @@
-import logo from './logo.svg';
 import './App.css';
+import Header from './Header';
+import Navbar from './Navbar';
+import { Routes,Route } from 'react-router-dom';
+import Articles from './Articles';
+import Topics from './Topics';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <div className="app">
+    <Header />
+    <Navbar />
+
+    <div className="main">
```

Comment:
Maybe a semantic HTML tag such as <main> would be more appropiate here?

Reply:
You mean <main> tag here?

Reply:
main tag